### PR TITLE
feat: add validator operator aliases to default multisig config

### PR DESCRIPTION
**Pull request by paulbalaji**
### Description

feat: add validator operator aliases to default multisig config

tldr:
updated this type and anywhere it's been used
image

- updates `MultisigConfig.validators` type from Address[] to ValidatorConfig[]
- `ValidatorConfig` includes both an address + an alias, to be able to track the operator aliases in-code
- update any usage of `defaultMultisigConfigs` and `MultisigConfig` type
- add `multisigConfigToIsmConfig` helper method to easily convert to `MultisigIsmConfig` format (ie. no aliases)

this will simplify the logic in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4774, and enable us to auto-generate the default ISMs docs page after an SDK update

### Testing

- added extra tests to ensure address/aliases are somewhat sound
- `yarn tsx scripts/check/check-validator-announce.ts -e mainnet3` still looks good

### CODE REVIEW

File: .changeset/four-meals-cheat.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@hyperlane-xyz/cli': major
+'@hyperlane-xyz/sdk': major
```
Comment (paulbalaji) [inline .changeset/four-meals-cheat.md:3]:
```suggestion
'@hyperlane-xyz/cli': minor
'@hyperlane-xyz/sdk': minor
```

**Review by tkporter — APPROVED** [review]
